ledger-api/grpc-definitions/com/daml/ledger/api/v1/transfer_command.proto

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+
+option java_outer_classname = "TransferCommandOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// This message is a WIP and originates from commands.proto
+// TODO reindex
+message TransferCommand {
+  // TODO deprecated, consider dropping it for 3.0
+  // Must correspond to the ledger ID reported by the Ledger Identification Service.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  // Optional
+  string ledger_id = ;
+
+  // Identifier of the on-ledger workflow that this command is a part of.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  // Optional
+  string workflow_id = ;
+
+  // Uniquely identifies the application or participant user that issued the command.
+  // Must be a valid ApplicationIdString (as described in ``value.proto``).
+  // Required unless authentication is used with a user token or a custom token specifying an application-id.
+  // In that case, the token's user-id, respectively application-id, will be used for the request's application_id.
+  string application_id = ;
+
+  // Uniquely identifies the command.
+  // The triple (application_id, party + act_as, command_id) constitutes the change ID for the intended ledger change,
+  // where party + act_as is interpreted as a set of party names.
+  // The change ID can be used for matching the intended ledger changes with all their completions.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  // Required
+  string command_id = ;
```

Comment:
From @andreaslochbihler-da :
In general, does it make sense to deduplicate transfer commands against transaction commands (and transfer-outs against transfer-ins)? It's not clear to me that there's a use-case for that. In that case, I'd recommend to change the triple into something that contains the kind of command.

Reply:
I'd suggest to wait with such a split. Command-dedup is a versatile tool, whose full set of use-cases we have not understood. There might be cases where we'd want to dedup a create vs. a transfer-out ... I don't know yet.